doc/man5/flux-config-accounting.rst

```diff
@@ -0,0 +1,36 @@
+=========================
+flux-config-accounting(5)
+=========================
+
+
+DESCRIPTION
+===========
+
+The flux-accounting priority plugin can be configured to assign different
+weights to the different factors used when calculating a job's priority.
+Assigning a higher weight to a factor will result in it having more
+influence in the calculated priority for a job.
```

Comment:
Maybe this should mention the default weight for factors that are not set in the config file?

Reply:
Good suggestion, thanks. I've just force-pushed up a fix that adds a table which lists the default weights for factors if they are not configured.